### PR TITLE
Swagger documentation split into 3 parts

### PR DESCRIPTION
**Pull request by AlexeyShok**
Swagger documentation split into 3 parts: for agent app, for admin app and common

### CODE REVIEW

**Review by DFSko — APPROVED** [review]

File: Api/Infrastructure/ApiExplorerGroupPerVersionConvention.cs [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    {
+        public void Apply(ControllerModel controller)
+        {
+            var controllerNamespace = controller.ControllerType.Namespace;
+            var controllerPurpose = controllerNamespace.Split('.').Last();
+
+            switch (controllerPurpose)
+            {
+                case "AdministratorControllers":
+                    controller.ApiExplorer.GroupName = "admin-v1.0";
```
Comment (kirillta) [inline Api/Infrastructure/ApiExplorerGroupPerVersionConvention.cs:16]: Why include version in a group name?

File: Api/Infrastructure/ApiExplorerGroupPerVersionConvention.cs [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+                    return;
+                case "AgentControllers":
+                    controller.ApiExplorer.GroupName = "agent-v1.0";
+                    return;
+                case "Controllers":
+                    controller.ApiExplorer.GroupName = "common-v1.0";
```
Comment (kirillta) [inline Api/Infrastructure/ApiExplorerGroupPerVersionConvention.cs:22]: I prefer `service`, not `common`

File: Api/Startup.cs [hunk 3]
```diff
@@ -104,7 +104,9 @@ public void ConfigureServices(IServiceCollection services)
 
             services.AddSwaggerGen(options =>
             {
-                options.SwaggerDoc("v1.0", new OpenApiInfo {Title = "HappyTravel.com Edo API", Version = "v1.0"});
+                options.SwaggerDoc("agent-v1.0", new OpenApiInfo {Title = "HappyTravel.com Edo API for Agent App", Version = "v1.0"});
```
Comment (kirillta) [inline Api/Startup.cs:107]: `Happytravel.com` please

File: Api/Startup.cs [hunk 4]
```diff
@@ -206,7 +209,9 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env, ILoggerF
             app.UseSwagger()
                 .UseSwaggerUI(options =>
                 {
-                    options.SwaggerEndpoint("/swagger/v1.0/swagger.json", "HappyTravel.com Edo API");
+                    options.SwaggerEndpoint("/swagger/agent-v1.0/swagger.json", "HappyTravel.com Edo API for Agent App");
```
Comment (kirillta) [inline Api/Startup.cs:212]: `/swagger/agent/v1.0/swagger.json` is more consistent as for me

File: Api/Startup.cs (hunk 3 above)
Comment (kirillta) [reply Api/Startup.cs:107]: And _for an agent app_. Technically it's not limited by one client

File: Api/Infrastructure/ApiExplorerGroupPerVersionConvention.cs (hunk 1 above)
Comment (AlexeyShok) [reply Api/Infrastructure/ApiExplorerGroupPerVersionConvention.cs]: Removed

File: Api/Startup.cs (hunk 3 above)
Comment (AlexeyShok) [reply Api/Startup.cs]: Renamed

File: Api/Startup.cs (hunk 4 above)
Comment (AlexeyShok) [reply Api/Startup.cs]: Renamed

File: Api/Infrastructure/ApiExplorerGroupPerVersionConvention.cs (hunk 2 above)
Comment (AlexeyShok) [reply Api/Infrastructure/ApiExplorerGroupPerVersionConvention.cs]: Renamed

**Review by kirillta — APPROVED** [review]

**Pull request merged**